### PR TITLE
Update Jenkinsfile for BRD website deploy

### PR DESCRIPTION
**Pull request by edpaget**
Remove production deploys from vets-website Jenkins file so they can be done using the BRD cycle like other components. 

Depends on https://github.com/department-of-veterans-affairs/devops/pull/2357

### CODE REVIEW

**Comment by edpaget** [comment]
I think should be all set.

File: Jenkinsfile [hunk 1]
```diff
@@ -211,30 +217,24 @@ node('vetsgov-general-purpose') {
...
+          build job: 'deploys/vets-website-dev', parameters: [
+            booleanParam(name: 'notify_slack', value: true),
+            stringParam(name: 'ref', value: commit),
+          ], wait: false
+        }
+      } else if (env.BRANCH_NAME == stagingBranch) {
+        steps {
+          build job: 'deploys/vets-website-staging', parameters: [
+            booleanParam(name: 'notify_slack', value: true),
+            stringParam(name: 'ref', value: commit),
```
Comment (wyattwalter) [inline Jenkinsfile]: I might be missing something, but does `commit` get set?

Comment (edpaget) [reply Jenkinsfile]: Oops that section was definitely messed up.

**Review by wyattwalter — APPROVED** [review]
LGTM!

**Pull request merged**